### PR TITLE
Docs: update user guide according to Dr Wee's comments

### PR DESCRIPTION
**Pull request by Zer0Legion**
**Issues requiring review by you guys**
1. **Be more specific in what the UG aims to do - a one-liner is not sufficient**- I have changed it, but I need more input from you guys to further improve the introduction. 
2. Whitespaces should be used to separate items more distinctly
3. Use diff text colours to better word the information
4. Add more info on what our app does
5. Better use of colours

**Issues unresolved:**
1. Can easily convert UG to PDF, but if changes are made it's not contrasted- not urgent.
2. Screenshots can highlight aesthetic parts- not urgent; do after revamp UI
6. Screenshot should show the full information and not cut off the long text
7. Need a table of contents

**Issues resolved:**
1. Use cases might be too technical a term
3. 'GUI similar to the below' is grammatically wrong
8. CLI is not very useful (Bold something else head tutor maybe)
9. Copying instead of copy
10. Make sure that the titles are consistent -> ____ing
11. Explain what help does in greater detail
12. What does “adds a person” mean
13. Explain the example
14. Shown as followed -> use a friendlier language
15. Description seems clinical
16. Putting the parameter summary at the bottom is fine
17. Hyperlinks are good, use more if necessary (but don’t put too many it can get confusing)
18. Place labels on images to link the image to examples
19. Some examples arent explained
20. FAQ -> Ccomputer
21. Do we need all the egs (4eg for 1 command)
22.  Number examples
23.  Examples can be bolded
24.  Box up lengthy information
25. Use “you” more in the features section

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -6,13 +6,19 @@
...
+However, its functions can also be extended to you if you are a head tutor of another course. 
 
-Our user guide aims to aid users in understanding how to get started with the application.
+Our user guide aims to help you understand how to get started with the application.
+
+
+We will cover the features in a chronological order, from:
+1. Installation, to 
+2. Simple commands like adding and editing, followed by 
+3. More advanced commands like filtering and exporting data.
```
Comment (delishad21) [inline docs/UserGuide.md:21]: Good inclusion

File: docs/UserGuide.md [hunk 2]
```diff
@@ -43,46 +50,78 @@ Our user guide aims to aid users in understanding how to get started with the ap
...
+I provide you with tips to use our app more effectively.
+</box>
+
+<box type="warning" seamless>
+
+**Caution:**
+I provide you with warnings about potential issues you might face.
+</box>
+
+
```
Comment (delishad21) [inline docs/UserGuide.md:77]: this legend labelling is good.

**Review by delishad21 — COMMENTED** [review]
LGTM.

**Review by danielleloh — COMMENTED** [review]
All changes look good to me :)

File: docs/UserGuide.md (hunk 1 above)
Comment (Pughal77) [reply docs/UserGuide.md:21]: Nice our aim for UG is much detailed now

**Review by Pughal77 — COMMENTED** [review]
Nice that you have included the suggestions made by Dr Wee

**Pull request merged**